OpenEdXMobile/src/main/java/org/edx/mobile/whatsnew/WhatsNewFragment.java

```diff
@@ -115,6 +121,40 @@ public void onPageScrollStateChanged(int state) {
         }
     }
 
+    public List<WhatsNewModel> getWhatsNewMessages(@NonNull String currBuildVersion,
```

Comment:
This function doesn't seem to be a fit in this class, this and `filterAndroidMessages` should be moved into a Util class making their tests easy to write too.